features/importers/myhelsinki_places/tests/test_import_images.py

```diff
@@ -106,10 +107,13 @@ def test_image_is_deleted(requests_mock, importer, places_response):
 
     Most likely it means that the image not available on the previous url.
     """
+    ImageFactory(
```

Comment:
what about the test above, i.e. `test_delete_image_with_wrong_new_license`? it also runs `importer.import_features()` twice.

Reply:
Fair point, I'll refactor